tests/lora/test_lora_layers_flux.py

```diff
@@ -145,3 +153,87 @@ def test_with_alpha_in_state_dict(self):
             "Loading from saved checkpoints should give same results.",
         )
         self.assertFalse(np.allclose(images_lora_with_alpha, images_lora, atol=1e-3, rtol=1e-3))
+
+
+@slow
+@require_torch_gpu
+@require_peft_backend
+class FluxLoRAIntegrationTests(unittest.TestCase):
```

Comment:
Can add a skip tag like we have for the Flux pipeline tests:
https://github.com/huggingface/diffusers/blob/2454b98af4a8a3ac5f3aedb641080c5a4f61f6dd/tests/pipelines/flux/test_pipeline_flux.py#L224